ragna/deploy/_ui/components/file_uploader.py

```diff
@@ -5,13 +5,17 @@
 from panel.reactive import ReactiveHTML
 from panel.widgets import Widget
 
+from .. import styles as ui
+
 
 class FileUploader(ReactiveHTML, Widget):  # type: ignore[misc]
     allowed_documents = param.List(default=[])
     allowed_documents_str = param.String(default="")
 
     file_list = param.List(default=[])
 
+    height_upload_container = param.String(default=ui.FILE_CONTAINER_HEIGHT)
```

Comment:
The [modal resizing functionality](https://github.com/Quansight/ragna/blob/ef6c6a5267315caac7b4646d0bca1938b7f9439a/ragna/deploy/_ui/js.py#L12-L15) has been broken on `main` for a while (try expand the 'Advanced Configuration' and see what happens).

I spent some time trying to get it to work, but in the end the only solution I could get to work was by introducing another `param` object for the height of the upload container. This parameter is watched, and when the Advanced Configuration card is expanded, this parameter is updated to a smaller value. See also `modal_configuration.py`.